Documentation/guides/tracing.md

```diff
@@ -98,6 +98,84 @@ And the output files should be found in the current directory. You can
 use the `-o` switch if you would prefer to output them to a specific
 directory.
 
+## How to get GC memory dumps?
+
+If running on desktop, you can use the `dotnet-gcdump` global tool.
+This can be installed via:
+
+```dotnetcli
+$ dotnet tool install --global dotnet-gcdump
+```
+
+To use it, for example:
+
+```
+# `hw-readline` is a standard Hello World, with a `Console.ReadLine()` at the end
+$ dotnet run --project hw-readline.csproj
+Hello, World!
+Press <ENTER> to continue
+
+# Then from another shell...
+
+# Determine which process ID to dump
+$ ps | grep 'dotnet.*hw-re'
+33972 ttys049    0:01.86 dotnet run --project hw-readline.csproj
+33977 ttys050    0:00.00 grep dotnet.*hw-re
+
+# Collect the GC info
+$ dotnet gcdump collect --process-id 33972
+Writing gcdump to '.../hw-readline/20230314_113922_33972.gcdump'...
+	Finished writing 5624131 bytes.
+```
+See the [`dotnet-gcdump` documentation][dotnet-gcdump]
+for further details about its usage.
+
+This will connect to a process and save a `*.gcdump` file. You can
+open this file in Visual Studio on Windows, for example:
+
+![Visual Studio GC Heap Dump](../images/VS-GC-Dump.png)
+
+To get this data from an Android application, you need all the above
+setup for `adb shell`, `dsrouter`, etc. except you need to change the
+provider for `dotnet-trace`:
+
+```
+$  dotnet-trace collect --diagnostic-port /tmp/maui-app --providers Microsoft-DotNETRuntimeMonoProfiler:0xC900003:4
```

Comment:
```suggestion
$  dotnet-trace collect --diagnostic-port /tmp/maui-app --providers Microsoft-DotNETRuntimeMonoProfiler: 0xC900001:4
```

(since you updated the text below)